stages/org.osbuild.grub2

```diff
@@ -38,8 +42,41 @@ and accompanying data can be installed from the built root via `uefi.install`.
 Both UEFI and Legacy can be specified at the same time.
 """
 STAGE_OPTS = """
-"required": ["root_fs_uuid"],
+"oneOf": [{
+  "required": ["root_fs_uuid"]
+}, {
+  "required": ["rootfs"]
+}],
+"definitions": {
+  "filesystem": {
+    "description": "Description of how to locate the a file system",
+    "type": "object",
+    "oneOf": [{
+      "required": ["uuid"]
+    }, {
+      "required": ["label"]
+    }],
+    "properties": {
+      "label": {
+        "description": "Identify the file system by label",
+        "type": "string"
+      },
+      "uuid": {
+        "description": "Identify the file system by UUID",
+        "type": "string",
+        "oneOf": [
+          { "pattern": "^[0-9A-Za-z]{8}(-[0-9A-Za-z]{4}){3}-[0-9A-Za-z]{12}$",
+            "examples": ["9c6ae55b-cf88-45b8-84e8-64990759f39d"] },
+          { "pattern": "^[0-9A-Za-z]{4}-[0-9A-Za-z]{4}$",
+            "examples": ["6699-AFB5"] }
```

Comment:
Wow. I would have been fine with `string`, but this works as well.

Reply:
Not my doing, c&p from another place in `osbuild`. I think it was done by Will.